crates/meroctl/src/cli/context/create.rs

```diff
@@ -31,18 +36,16 @@ struct ListApplicationsResponse {
 }
 
 #[derive(Debug, Parser)]
-#[group(required = true)]
```

Comment:
does this mean `meroctl context create` now works?

by "works" I mean, `clap` won't report an issue

Reply:
Yes, I had some weird issue that I couldn't figure out, so I just put application_id as exclusive so it keeps the same functionality and it fixes the issue.